demo/docs/3d_human_pose_demo.md

```diff
@@ -36,3 +36,49 @@ python demo/body3d_two_stage_img_demo.py \
     --rebase-keypoint-height \
     --show-ground-truth
 ```
+
+### 3D Human Pose Two-stage Estimation Image Demo
+
+#### Using mmdet for human bounding box detection and top-down model for the 1st stage (2D pose detection), and inference the 2nd stage (2D-to-3D lifting)
+
+Assume that you have already installed [mmdet](https://github.com/open-mmlab/mmdetection).
+
+```shell
+python demo/body3d_two_stage_video_demo.py \
+    ${MMDET_CONFIG_FILE} \
+    ${MMDET_CHECKPOINT_FILE} \
+    ${MMPOSE_CONFIG_FILE_STAGE_1} \
+    ${MMPOSE_CONFIG_FILE STAGE_1} \
+    ${MMPOSE_CONFIG_FILE_STAGE_2} \
+    ${MMPOSE_CONFIG_FILE STAGE_2} \
```

Comment:
CKPT

Reply:
How about naming it as something like 'MMPOSE_CONFIG_FILE_2D' and 'MMPOSE_CONFIG_FILE_3D'

Reply:
Yes, I think that will be more clear.